molecule/default/molecule-no-systemd.yml

```
@@ -16,13 +16,12 @@ lint: |
   ansible-lint
   flake8
 platforms:
-  - image: amazonlinux:2
-    name: amazonlinux2
-    platform: amd64
-  - dockerfile: Dockerfile_debian_9.j2
-    image: debian:stretch-slim
-    name: debian9
-    platform: amd64
+  # Amazon Linux 2023 does not offer the following packages required by httpd:
+  # mod_auth_gssapi,mod_authnz_pam
```

We usually put a space after a comma- slightly more readable:
```suggestion
  # mod_auth_gssapi, mod_authnz_pam
```

---

59d439c has all relevant changes